src/runtime/gecko/util.js

```diff
@@ -0,0 +1,41 @@
+export { documentReady, getResourceLinks, getMeta } from '../web/util';
+import { getLangRevisionMap, getLangRevisionTuple } from '../web/util';
+
+export function getXULResourceLinks(doc) {
+  return Array.prototype.map.call(
+    doc.querySelectorAll('messagebundle'),
```

Comment:
I'm not sure if it'll fly with people to use add another NS to every XUL file for the sake of our internal codebase size